scipy/spatial/ckdtree/src/ckdtree_globals.cxx

```diff
@@ -0,0 +1,7 @@
+#define NPY_NO_DEPRECATED_API NPY_API_VERSION
```

Comment:
I've been playing with this deprecation mechanism a bit lately. It's a little dangerous to do this; if you're using a macro or function that gets deprecated in Numpy 1.10 then this will break the build. So better to use ``NPY_1_9_API_VERSION``.